### ISSUE TITLE
Admin users list should not provide bulk remove checkbox for user with negative spam score

### ISSUE THREAD

**Issue by Firefishy** [openstreetmap/openstreetmap-website#5097]
### Problem

The [admin users list](https://www.openstreetmap.org/users/active) has a checkbox ☑️ for each user in the list. The admin user list is primarily used for bulk removing ("hide") spam accounts.

It is very easy to accidentally tick an active mapper and remove ("hide") their account using this feature. The number of edits or "Spam Score" is not displayed on the user list to give the admin operator feedback.

### Description

Ideally mappers with edits or a negative spam score **should not be given a bulk user remove checkbox** in the admin user list to avoid accidentally allowing bulk removal of their account.

Mappers accounts could still be removed by an admin via the user's individual profile page.

**Comment by AntonKhorev** [comment]
> Ideally mappers with edits or a negative spam score should not be given a bulk user remove checkbox in the admin user list

What you actually want is a spam score filter and changeset count filter similar to existing ip address filter, right?

**Comment by Firefishy** [comment]
> > Ideally mappers with edits or a negative spam score should not be given a bulk user remove checkbox in the admin user list
> 
> What you actually want is a spam score filter and changeset count filter similar to existing ip address filter, right?

Yes, that could achieve the same result. Or just remove the checkbox for users with negative spam score or edits.

If done as a filter then could the filter be retained after submission. eg: after clicking "Hide Selected Users"

**Comment by AntonKhorev** [comment]
What are the typical spam scores of users you see in the list?

**Comment by Firefishy** [comment]
> What are the typical spam scores of users you see in the list?

Normally >= 10.
